client/src/components/messenger/controllers/Contacts.js

```diff
@@ -0,0 +1,69 @@
+export const ContactsController = {
+    getContacts: async token => {
+        // make request for the contacts of user and wait for the json response
+        const data = await (await fetch("/api/contacts", {
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json",
+                "Authorization": `Bearer ${token}`,
+            },
+        })).json();
+
+        // get the list of contacts if successful
+        return data ? data.contactList : [];
+    },
+    evaluateStatus: lastActive => {
+        // for now status is either online or offline
+        // if user not active within the last 5mins - offline
+        let diff = Date.now() - (new Date(lastActive)).getTime();
+        return diff > (5 * 60 * 1000) ? "offline": "online";
+    },
+    addContact: async ( token, email ) => {
```

Comment:
nit: Format document?

```suggestion
    addContact: async (token, email) => {
```